fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/keyspace/ResolverValidator.java

```diff
@@ -75,18 +77,22 @@ public static void validate(@Nullable final FDBStoreTimer timer,
                                 final int reverseLookupPipelineSize,
                                 final boolean badEntriesOnly,
                                 @Nonnull final Consumer<ValidatedEntry> entryListener) {
-        try (RecordCursor<ValidatedEntry> cursor = new AutoContinuingCursor<>(
-                resolver.getDatabase().newRunner(),
-                (context, continuation) ->
-                    validate(resolver, context, continuation, reverseLookupPipelineSize, false,
-                            new ScanProperties(executeProperties.build())),
-                3)) {
-            resolver.getDatabase().asyncToSync(timer, FDBStoreTimer.Waits.WAIT_VALIDATE_RESOLVER,
-                    cursor.forEach(validatedEntry -> {
-                        if (!badEntriesOnly || validatedEntry.getValidationResult() != ValidationResult.OK) {
-                            entryListener.accept(validatedEntry);
-                        }
-                    }));
+        try (FDBDatabaseRunner runner = resolver.database.newRunner()) {
+            runner.setContextConfigBuilder(FDBRecordContextConfig.newBuilder()
+                    .setSaveOpenStackTrace(true));
```

Comment:
Is this intentional? It seems like we wouldn't want to always do this.

Reply:
Whoops, yeah. I turned this on during some debugging to look into "transaction already closed errors", but we don't need this any more